lib/matplotlib/widgets.py

```diff
@@ -1010,20 +1011,25 @@ def __init__(self, ax, label, initial='',
             The color of the box when the mouse is over it.
         label_pad : float
             The distance between the label and the right side of the textbox.
+        textalignment : {'left', 'center', 'right'}
+            The horizontal location of the text.
         """
         super().__init__(ax)
 
-        self.DIST_FROM_LEFT = .05
```

Comment:
This should be deprecated instead of directly removed.

Reply:
How should I do it? something like `_api.deprecated("3.4")property(self lambda : self.DIST_FROM_LEFT)` at the beginning of the class afnd without removing self.DIST_FROM_LEFT?

Reply:
The lambda (and `__init__`) will have to access a different variable (`self._DIST_FROM_LEFT`), or it would be recursive.

Reply:
Thanks for the help!